config/application.rb

```diff
@@ -26,5 +26,6 @@ class Application < Rails::Application
       generate.routing_specs false
       generate.controller_specs false
     end
+    config.stripe.publishable_key = ENV['PUBLISHABLE_KEY']
```

Comment:
You need to set this to be able to use stripe.js. And we set the tests to publishable in all environments.